<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>567</width>
     <height>379</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="graphicTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Cassiniwidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="title">
        <string>Cassini</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="CassiniGrid">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="2" column="0">
           <widget class="QGroupBox" name="DepthGroupBox">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Depth</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <layout class="QHBoxLayout" name="cassiniDepthHorizontalLayout">
               <item>
                <widget class="QLabel" name="cassiniDepthLabel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Depth</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="cassiniDepthSpinBox">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>5</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2" colspan="5">
           <widget class="CassiniWidget" name="Cassini" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QGroupBox" name="rGroupBox">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="title">
             <string>R value</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <layout class="QHBoxLayout" name="rHorizontalLayout">
               <item>
                <widget class="QLabel" name="rLabel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>R</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QLineEdit" name="rEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="rSlider">
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="pageStep">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                 <property name="tickInterval">
                  <number>5</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Curvewidget">
       <attribute name="title">
        <string>Curve</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="curveGridLayout" rowstretch="0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="0" column="0" rowspan="2" colspan="13">
           <widget class="CurveWidget" name="Curve" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>547</width>
              <height>175</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QGroupBox" name="curveDepthGroupBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>85</height>
             </size>
            </property>
            <property name="title">
             <string>Depth</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="curveDepthHorisontalLayout">
               <item>
                <widget class="QLabel" name="curveDepthLabel">
                 <property name="text">
                  <string>Depth :</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QSpinBox" name="curveDepthSpinBox">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="3" colspan="4">
           <widget class="QGroupBox" name="scaleGroupBox">
            <property name="minimumSize">
             <size>
              <width>154</width>
              <height>85</height>
             </size>
            </property>
            <property name="title">
             <string>Scale</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="scaleVerticalLayout">
               <item>
                <layout class="QHBoxLayout" name="xScaleHorizontalLayout">
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLabel" name="xScaleLabel">
                   <property name="text">
                    <string>xScale : </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLineEdit" name="xScaleEdit">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="yScaleHorizontalLayout">
                 <item>
                  <widget class="QLabel" name="yScaleLabel">
                   <property name="text">
                    <string>YScale : </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLineEdit" name="yScaleEdit">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="7" colspan="4">
           <widget class="QGroupBox" name="tickSizesGroupBox">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>85</height>
             </size>
            </property>
            <property name="title">
             <string>Tick sizes</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="tickSizesVerticalLayout">
               <item>
                <layout class="QHBoxLayout" name="xTickSizesHorizontalLayout">
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLabel" name="xTickLabel">
                   <property name="text">
                    <string>XTick : </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLineEdit" name="xTickEdit">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="yTickSizesHorizontalLayout">
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLabel" name="yTickLabel">
                   <property name="text">
                    <string>YTick : </string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QLineEdit" name="yTickEdit">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="11" alignment="Qt::AlignRight">
           <widget class="QPushButton" name="resetScaleButton">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Reset Scale</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurveWidget</class>
   <extends>QWidget</extends>
   <header location="global">curvewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CassiniWidget</class>
   <extends>QWidget</extends>
   <header location="global">cassiniwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
